bin/commands/init/mvs/index.ts

```diff
@@ -0,0 +1,138 @@
+/*
+  This program and the accompanying materials are made available
+  under the terms of the Eclipse Public License v2.0 which
+  accompanies this distribution, and is available at
+  https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as xplatform from 'xplatform';
+import * as fs from '../../../libs/fs';
+import * as shell from '../../../libs/shell';
+import * as zoslib from '../../../libs/zos';
+import * as zosJes from '../../../libs/zos-jes';
+import * as zosdataset from '../../../libs/zos-dataset';
+import * as common from '../../../libs/common';
+import * as config from '../../../libs/config';
+import * as stringlib from '../../../libs/string';
+
+export function execute(allowOverwrite?: boolean) {
+  common.printLevel1Message(`Initialize Zowe custom data sets`);
+  common.requireZoweYaml();
+  const ZOWE_CONFIG = config.getZoweConfig();
+  
+  const datasets = ['parmlib', 'authLoadlib', 'authPluginLib'];
+
+  const prefix = ZOWE_CONFIG.zowe.setup?.dataset?.prefix;
+  if (!prefix) {
+    common.printErrorAndExit(`Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file.`, undefined, 157);
```

Comment:
This error is produced only if `prefix` is missing in `zowe.yaml` **AND** `defaults`!

Reply:
Handled by removing jcllib from defaults.yaml and forcing user to do it in their own yaml as it was before.